Lib/pathlib.py

```diff
@@ -1176,6 +1176,25 @@ def write_text(self, data, encoding=None, errors=None):
                             data.__class__.__name__)
         with self.open(mode='w', encoding=encoding, errors=errors) as f:
             return f.write(data)
+    
+    def append_bytes(self, data):
+        """
+        Open the file in bytes mode, write to it at its end, and close the file.
+        """
+        # type-check for the buffer interface before truncating the file
+        view = memoryview(data)
+        with self.open(mode='ab') as f:
+            return f.write(view)
+    
+    def append_text(self, data, encoding=None, errors=None):
+        """
+        Open the file in bytes mode, write to it at its end, and close the file.
+        """
+        if not isinstance(data, str):
+            raise TypeError('data must be str, not %s' %
+                            data.__class__.__name__)
+        with self.open(mode='a', encoding=encoding, errors=errors) as f:
```

Comment:
shouldn't the `mode` be set explicitly to `ta`?

Reply:
I don't think so. Text mode is the default mode. `write_text` and `read_text` doesn't set it either.